### DISCUSSION TITLE
[Feature] List of assets not backed up on mobile

### DISCUSSION THREAD

**Discussion by Krolitian** [immich-app/immich#8506] [Feature Request]
### The feature

When trying to use this as a backup solution, it's frustrating not knowing out of tens of thousands of assets the few that aren't backed up.

I recommend a button/tab on the Backup page that displays all files identified in the phone's filesystem but not yet backed up, whether due to corruption, file type not supported, or if it's still queued so that we can diagnose the issue and get it backed up one way or another.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by OliviaD2001** [comment]
This would be incredibly useful!

**Comment by Ccheese4** [comment]
concur with this. Maybe clicking on the "Remainder" box to bring it up? The issue I'm running into is that I have backed up my photos to Immich and deduplicated on the server, but the app is wanting to back up items and reupload items that were removed during deduplication. Adding this feature would allow me to delete the duplicate items from my phone so that the don't keep trying to back up. 
An alternate would be to save the hash and provide the user the option to not reupload deduplicated items, but I ran across another thread saying that this was acting as expected for now (https://github.com/immich-app/immich/issues/10873).

**Comment by bo0tzz** [comment]
Somehow got missed for a while but this is a dupe of #2155
